core/src/main/java/org/testcontainers/DockerClientFactory.java

```diff
@@ -17,9 +17,6 @@
 import lombok.SneakyThrows;
 import lombok.Synchronized;
 import lombok.extern.slf4j.Slf4j;
-import org.hamcrest.BaseMatcher;
-import org.hamcrest.Description;
-import org.rnorth.visibleassertions.VisibleAssertions;
```

Comment:
Using `VisibleAssertions` for the logging in checks seems like major overkill, so I've changed it to use simple log statements.